docs/app/partials/layout-options.tmpl.html

```diff
@@ -97,77 +122,36 @@
       </div>
     </demo-file>
   </docs-demo>
+
   <p>
-    <code>layout-wrap</code> allows <code>flex</code> children to wrap within the container if the elements use more than 100%.
+    <code>layout-wrap</code> allows <code>flex</code> children to wrap within the container if the elements use more
+    than 100%.
     <br/>
   </p>
 
   <br/>
 
   <br/>
+    <hr>
+    <br/>
 
-  <docs-demo demo-title="Responsive Flex Attributes" class="small-demo colorNested-noPad">
-    <demo-file name="index.html">
-      <div layout="row">
-        <div flex="66" flex-sm="33">
-          I flex to one-third of the space on mobile, and two-thirds on other devices.
-        </div>
-        <div flex="33" flex-sm="66">
-          I flex to two-thirds of the space on mobile, and one-third on other devices.
-        </div>
-      </div>
-    </demo-file>
-  </docs-demo>
-
-  <p>
-    See the <a href="layout/grid">Layout System</a> page for a basic explanation
-    of flex and offset attributes.
-  </p>
-
-  <table class="md-api-table">
-    <tr>
-      <td>flex</td>
-      <td>Sets flex.</td>
-    </tr>
-    <tr>
-      <td>flex-sm</td>
-      <td>Sets flex on devices less than 600px wide.</td>
-    </tr>
-    <tr>
-      <td>flex-gt-sm</td>
-      <td>Sets flex on devices greater than 600px wide.</td>
-    </tr>
-    <tr>
-      <td>flex-md</td>
-      <td>Sets flex on devices between 600px and 960px wide..</td>
-    </tr>
-    <tr>
-      <td>flex-gt-md</td>
-      <td>Sets flex on devices greater than 960px wide.
-    </tr>
-    <tr>
-      <td>flex-lg</td>
-      <td>Sets flex on devices between 960px and 1200px.</td>
-    </tr>
-    <tr>
-      <td>flex-gt-lg</td>
-      <td>Sets flex on devices greater than 1200px wide.</td>
-    </tr>
-  </table>
+    <h3>Show &amp; Hide </h3>
 
-  <br/>
+  <p>Use the <code>show</code> <code>hide</code> APIs to responsively show or hide elements</p>
```

Comment:
Might want to mention that these work like `ng-show` and `ng-hide` (vs. `ng-if`)